### PR TITLE
added Java client

### PR DESCRIPTION
**Pull request by mpresteg**
Added Java client - swagger auto gen from client pom.xml - command line tools in freq-tools

### CODE REVIEW

File: client/java/freq-tools/pom.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.dishevelled</groupId>
+      <artifactId>dsh-compress</artifactId>
+      <version>1.1</version>
+      <scope>compile</scope>
+    </dependency>
+     <dependency>
+      <groupId>org.nmdp.ngs</groupId>
+      <artifactId>ngs-hml</artifactId>
```
Comment (fscheel) [inline client/java/freq-tools/pom.xml:78]: Do we need this dependency? I didn't see any reference to it.

File: client/java/pom.xml [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  		<version>3.0</version>
+  	</dependency>
+  	<dependency>
+  		<groupId>com.google.guava</groupId>
+  		<artifactId>guava</artifactId>
+  		<version>18.0</version>
+  	</dependency>
+  	<dependency>
+  		<groupId>org.nmdp.gl</groupId>
+  		<artifactId>gl-client-local</artifactId>
```
Comment (fscheel) [inline client/java/pom.xml:214]: This and the `gl-client` dependcy below are others I didn't see used. And I'm not sure about `inject` and `guava`.

File: client/java/freq-client/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
+# swagger-java-client
```
Comment (fscheel) [inline client/java/freq-client/README.md:1]: This file probably doesn't need to be checked in.

**Review by fscheel — COMMENTED** [review]
Didn't we want to add the `client/java/freq-client/[src,doc,gradle]` and some other files to gitignore?

File: client/java/freq-tools/pom.xml (hunk 1 above)
Comment (mpresteg) [reply client/java/freq-tools/pom.xml:78]: Wow - lots of garbage in there.  Thanks for catching.  I ran the maven dependency analyzer and was able to remove a bunch.

File: client/java/pom.xml (hunk 2 above)
Comment (mpresteg) [reply client/java/pom.xml:214]: Yep.  See above.

**Comment by mpresteg** [comment]
Closing to make repairs as suggested during review.

**Comment by fscheel** [comment]
You could have just pushed to your branch again and github would have added it to this PR

**Comment by mpresteg** [comment]
Good to know (for next time).  Thanks for the review.

**Pull request closed without merging**